Antidote/CallsManager.m

```diff
@@ -205,28 +214,46 @@ - (void)activeCallSpeakerButtonPressed:(ActiveCallViewController *)controller
 
 - (void)activeCallPauseButtonPressed:(ActiveCallViewController *)controller
 {
+    NSError *error;
     if (controller.pauseSelected) {
-        if ([self.manager sendCallControl:OCTToxAVCallControlResume toCall:self.currentCall error:nil]) {
+        if ([self.manager sendCallControl:OCTToxAVCallControlResume toCall:self.currentCall error:&error]) {
             controller.pauseSelected = NO;
         }
+        else {
+            DDLogWarn(@"%@, activeCallPauseButtonPressed:OCTToxAVCallControlResume error: %@", self, error.localizedFailureReason);
+        }
     }
     else {
         if ([self.manager sendCallControl:OCTToxAVCallControlPause toCall:self.currentCall error:nil]) {
             controller.pauseSelected = YES;
         }
+        else {
+            DDLogWarn(@"%@, activeCallPauseButtonPressed:OCTToxAVCallControlPause error: %@", self, error.localizedFailureReason);
+        }
     }
 }
 
 - (void)activeCallDeclineIncomingCallButtonPressed:(ActiveCallViewController *)controller
 {
-    [self.manager sendCallControl:OCTToxAVCallControlCancel toCall:self.pendingIncomingCall error:nil];
+    NSError *error;
```

Comment:
Here you can add verbose log as for `activeCallAnswerIncomingCallButtonPressed` method.